packages/mesos/build

```diff
@@ -32,37 +33,37 @@ cp /usr/lib/x86_64-linux-gnu/libsvn_delta-1.so.1 "$libdir"
 cp /usr/lib/x86_64-linux-gnu/libsvn_subr-1.so.1 "$libdir"
 cp /usr/lib/x86_64-linux-gnu/libapr-1.so.0 "$libdir"
 cp /usr/lib/x86_64-linux-gnu/libaprutil-1.so.0 "$libdir"
+cp /usr/lib/x86_64-linux-gnu/libdb-5.3.so "$libdir"
 
 export PKG_PATH=$PKG_PATH
 systemd_master="$PKG_PATH"/dcos.target.wants_master/dcos-mesos-master.service
 mkdir -p "$(dirname "$systemd_master")"
 envsubst '$PKG_PATH' < /pkg/extra/dcos-mesos-master.service > "$systemd_master"
 
-systemd_slave="$PKG_PATH"/dcos.target.wants_slave/dcos-mesos-slave.service
-mkdir -p "$(dirname "$systemd_slave")"
-envsubst '$PKG_PATH' < /pkg/extra/dcos-mesos-slave.service > "$systemd_slave"
+systemd_agent="$PKG_PATH"/dcos.target.wants_agent/dcos-mesos-agent.service
+mkdir -p "$(dirname "$systemd_agent")"
+envsubst '$PKG_PATH' < /pkg/extra/dcos-mesos-agent.service > "$systemd_agent"
 
-systemd_slave_public="$PKG_PATH"/dcos.target.wants_slave_public/dcos-mesos-slave-public.service
-mkdir -p "$(dirname "$systemd_slave_public")"
-envsubst '$PKG_PATH' < /pkg/extra/dcos-mesos-slave-public.service > "$systemd_slave_public"
+systemd_agent_public="$PKG_PATH"/dcos.target.wants_agent_public/dcos-mesos-agent-public.service
+mkdir -p "$(dirname "$systemd_agent_public")"
+envsubst '$PKG_PATH' < /pkg/extra/dcos-mesos-agent-public.service > "$systemd_agent_public"
 
 # setup additonal volume service which discovers /dcos/volumeN mounts and creates an optional
 # EnvironmentFile that contains a MESOS_RESOURCES env variable. This MESOS_RESOURCES adds
 # Mount resources to another existing MESOS_RESOURCES variable, which means the EnvironmentFile
 # created by this service has to be the last
-public_volume_service="$PKG_PATH"/dcos.target.wants_slave_public/dcos-vol-discovery-pub-agent.service
+public_volume_service="$PKG_PATH"/dcos.target.wants_agent_public/dcos-vol-discovery-pub-agent.service
 mkdir -p "$(dirname ${public_volume_service})"
 envsubst '$PKG_PATH' < /pkg/extra/dcos-vol-discovery-pub-agent.service > "$public_volume_service"
 
-private_volume_service="$PKG_PATH"/dcos.target.wants_slave/dcos-vol-discovery-priv-agent.service
+private_volume_service="$PKG_PATH"/dcos.target.wants_agent/dcos-vol-discovery-priv-agent.service
 mkdir -p "$(dirname ${private_volume_service})"
 envsubst '$PKG_PATH' < /pkg/extra/dcos-vol-discovery-priv-agent.service > "$private_volume_service"
 
 disk_resource_script="$PKG_PATH/bin/make_disk_resources.py"
 cp /pkg/extra/make_disk_resources.py "$disk_resource_script"
 chmod +x "$disk_resource_script"
 
-# Copy the json file describing logrotate module.
-module_json="$PKG_PATH/etc/mesos-slave-modules/logrotate.json"
-mkdir -p "$(dirname "$module_json")"
-cp /pkg/extra/logrotate.json "$module_json"
+# Install agent modules.
+mkdir -p "$PKG_PATH/etc/mesos-agent-modules"
```

Comment:
keep this slave-modules for now. Should switch all the mesos config at once, not half of it here.